lib/tasks/analyse/links.rb

```diff
@@ -52,16 +52,15 @@ def write_overview_worksheet(results, spreadsheet)
     overview_worksheet = spreadsheet.worksheets[0]
     overview_worksheet.title = 'Overview'
 
-    add_results_to_worksheet(results, overview_worksheet)
+    add_results_to_worksheet(results, overview_worksheet, "accordion")
```

Comment:
Passing "accordion" into this method is confusing when we're trying to update the "overview" worksheet. I think this is because you're abusing your own `format_columns` method? This sort of approach is fragile, and quite obscure.

It might be a bit cleaner and more readable if you instead passed in an array of the columns you want in the worksheet? So in this case it would become:

```ruby
add_results_to_worksheet(results, overview_worksheet, column_names)
```